chart/env/prod.yaml

```diff
@@ -113,6 +113,19 @@ cacheMaintenance:
       cpu: 2
       memory: "8Gi"
 
+# --- cron jobs  ---
+metricsCollector:
+  action: "collect-metrics"
+  schedule: "*/2 * * * *"
+  # every two minutes
+  nodeSelector: {}
+  resources:
+    requests:
+      cpu: 0
```

Comment:
maybe limit the resources in production, depending on the need.

Reply:
It is going to be a light cron job (Will compute two aggregated queries from mongodb) is it ok to set limit-request to 1?